src/cli.rs

```diff
@@ -15,7 +15,24 @@ use timber::{Level, DEFAULT_LEVEL, LEVELS};
 use std::path::PathBuf;
 
 #[derive(Debug, Serialize, StructOpt)]
-#[structopt(name = "Rover", about = "The new CLI for Apollo", global_settings = &[structopt::clap::AppSettings::ColoredHelp])]
+#[structopt(name = "Rover", global_settings = &[structopt::clap::AppSettings::ColoredHelp], about = "
+Rover - Your Graph Companion
+Read the getting started guide: https://go.apollo/dev/rover-start
```

Comment:
is this right?

Reply:
This is just a shortlink that we can set up. Right now, it goes nowhere, but I also just realized it says `/dev` instead of `.dev` so no, it's not right 👎

Reply:
gotcha! i think i'm remembering now 👍